tests/flow/test_point.py

```diff
@@ -192,3 +192,39 @@ def test_nested_point(self):
         res = redis_graph.query(q)
         self.env.assertEquals(res.result_set[0][0], expected_value)
 
+    def test_point_coordinates(self):
+        # read latitude
+        expected_value = 32.070794860
```

Comment:
add test for
RETURN point({latitude:32.070794860, longitude:34.820751118}).v

Reply:
Ok. Test added in [64c7434](https://github.com/RedisGraph/RedisGraph/pull/2524/commits/64c7434e4b5346799eda972ea0bcb1b35a204146)